### PR TITLE
Add rootdisk size on compute offering

### PR DESCRIPTION
**Pull request by GabrielBrascher**
## Description

This PR allows configuring a Service Offering with root disk size.
PR [Allow to configure root disk size via Service Offering (disk offering of type Service). #4341](https://github.com/apache/cloudstack/pull/4341) allows enforcing VM Root disk size accordingly to its Service Offering.

## How Has This Been Tested?

- Create Service offering via UI with rootdisk (CloudStack from commit https://github.com/apache/cloudstack/pull/4341/commits/4f501f0624a44fe2414835f17f210bf977f08846)
- Assert that the offering has indeed the root disk size configured (same test as in https://github.com/apache/cloudstack/pull/4341)

## Screenshots:

1. **Empty:**
image

2. **Value not supported:**
image

3. **Value for Root disk size supported:**
image

### CODE REVIEW

**Comment by GabrielBrascher** [comment]
@blueorangutan package

File: src/views/offering/AddComputeOffering.vue [hunk 1]
```diff
@@ -333,6 +333,28 @@
             }]"
             :placeholder="this.$t('label.networkrate')"/>
         </a-form-item>
+          <a-form-item>
```
Comment (davidjumani) [inline src/views/offering/AddComputeOffering.vue:336]:
```suggestion
          <a-form-item v-if="apiParams.rootdisksize">
```

File: src/views/offering/AddComputeOffering.vue [hunk 2]
```diff
@@ -333,6 +333,28 @@
...
+            <span slot="label">
+              {{ $t('label.root.disk.size') }}
+              <a-tooltip :title="apiParams.provisioningtype.description">
```
Comment (davidjumani) [inline src/views/offering/AddComputeOffering.vue:339]:
```suggestion
              <a-tooltip :title="apiParams.rootdisksize.description">
```

**Comment by davidjumani** [comment]
@blueorangutan package

File: src/views/offering/AddComputeOffering.vue (hunk 2 above)
Comment (GabrielBrascher) [reply src/views/offering/AddComputeOffering.vue]: Thanks for the review @davidjumani, fixed.

File: src/views/offering/AddComputeOffering.vue (hunk 1 above)
Comment (GabrielBrascher) [reply src/views/offering/AddComputeOffering.vue]: Thanks :-)

**Comment by rhtyd** [comment]
@blueorangutan package

**Review by davidjumani — APPROVED** [review]
Code LGTM. Need to test once the ACS PR has been merged

**Pull request merged**